handlers/download_test.go

```diff
@@ -79,14 +79,18 @@ func (w errWriter) Write([]byte) (int, error) {
 func TestDownloadDoReturnsRedirect(t *testing.T) {
 	t.Parallel()
 	mockCtrl := gomock.NewController(t)
+	mockCollectionID := ""
+	mockDownloadToken := ""
+	mockServiceAuthToken := ""
+	mockUserAuthToken := ""
 	defer mockCtrl.Finish()
 
 	Convey("Given a public link to the download exists on the filter api then return a status 301 to the download", t, func() {
 		req := httptest.NewRequest("GET", "http://localhost:28000/downloads/filter-outputs/abcdefg.csv", nil)
 		w := httptest.NewRecorder()
 		r := mux.NewRouter()
 
-		fc := mocks.NewMockFilterClient(mockCtrl)
+		fc := mocks.NewMockFilterClient(mockCtrl) //f := NewFilter(mockRenderer, mockFilterClient, mockDatasetClient, nil, nil, nil, "", "","")
```

Comment:
delete commented out code please.

Reply:
Ah, I went looking for this before I PRed couldn't find it and thought I must have removed it already. Cheers for spotting.